linear-base.cabal

```diff
@@ -183,7 +183,7 @@ test-suite examples
         base,
         linear-base,
         tasty,
-        tasty-hedgehog,
+        tasty-hedgehog < 1.2,
```

Comment:
Yes, otherwise `cabal build` would fail.  I am using GHC 9.2.2.
I think this restriction can be lifted if the underlying problem is fixed:
```
Building test suite 'test' for linear-base-0.2.0..
[1 of 9] Compiling Test.Data.Destination ...

test/Test/Data/Destination.hs:22:7: error: [-Wdeprecations, -Werror=deprecations]
    In the use of ‘testProperty’ (imported from Test.Tasty.Hedgehog):
    Deprecated: "testProperty will cause Hedgehog to provide incorrect instructions for re-checking properties"
   |
22 |     [ testProperty "alloc . mirror = id" roundTrip,
   |       ^^^^^^^^^^^^
```
This deprecation was introduced in `tasty-hedgehog-1.2`.

Reply:
Added a comment to the .cabal file to explain why this bound is needed.